src/main/java/jakarta/validation/Validation.java

```diff
@@ -316,12 +316,11 @@ public Configuration<?> configure() {
 	private static class DefaultValidationProviderResolver implements ValidationProviderResolver {
 		@Override
 		public List<ValidationProvider<?>> getValidationProviders() {
-			// class loading and ServiceLoader methods should happen in a PrivilegedAction
 			return GetValidationProviderListAction.getValidationProviderList();
 		}
 	}
 
-	private static class GetValidationProviderListAction implements PrivilegedAction<List<ValidationProvider<?>>> {
+	private static class GetValidationProviderListAction {
```

Comment:
Since you've removed the `NewProviderInstance`, would it make sense to simply merge the impl of this action into  `DefaultValidationProviderResolver` and, by that, get rid of `GetValidationProviderListAction` as well?

Reply:
I have addressed this review comment in a second commit in the PR.  I considered it earlier, but with the use of INSTANCE, I didn't want to change the flow too much.  I believe I have done what is needed in order to address it.